### PR TITLE
typo and update

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/timezone-handling-changes.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ Anki 2.1.22+ and AnkiMobile 2.0.57+ contain an optional change to the way Anki c
 
 For most users, the new calculation should produce the same number of elapsed days as the old calculation. For some users, enabling this code may move Anki forward or back by one day, but this will only happen once, and Anki should be more robust against timezone changes afterwards.
 
-**Please make sure you are running the above-mentioned versions prior to enabling the new code.** If any of your Anki clients have not been updated yet, you won't be able to sync them while this feature is enabled. **AnkiDroid does not currently support these changes**.
+**Please make sure you are running the above-mentioned versions prior to enabling the new code.** If any of your Anki clients have not been updated yet, you won't be able to sync them while this feature is enabled. **AnkiDroid does only support these changes from version 2.1.15**.
```
Comment (dae) [inline src/timezone-handling-changes.md:7]:
```suggestion
**Please make sure you are running the above-mentioned versions prior to enabling the new code.** If any of your Anki clients have not been updated yet, you won't be able to sync them while this feature is enabled.
```

File: src/timezone-handling-changes.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ Anki 2.1.22+ and AnkiMobile 2.0.57+ contain an optional change to the way Anki c
```
Comment (dae) [inline src/timezone-handling-changes.md:4]: How about the previous paragraph is changed to read the following?

"Anki 2.1.22+, AnkiMobile 2.0.57+, and AnkiDroid 2.15+ contain"

Comment (cqg2) [reply src/timezone-handling-changes.md:4]: Yes, easier :)

**Comment by dae** [comment]
Thanks!

**Pull request merged**